### PR TITLE
Allow Symfony 3.

### PR DESCRIPTION
**Pull request by aitboudad**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -11,15 +11,15 @@
...
+        "symfony/console": "~2.3|~3.0",
+        "symfony/intl": "~2.3|~3.0"
     },
     "require-dev": {
-        "symfony/yaml": "~2.3",
-        "symfony/browser-kit": "~2.3",
-        "symfony/twig-bundle": "~2.3"
+        "symfony/yaml": "~2.3|~3.0",
+        "symfony/browser-kit": "~2.3|~3.0",
+        "symfony/twig-bundle": "~2.3|~3.0"
```
Comment (stof) [inline composer.json]: you should add the dev requirement on the phpunit bridge to catch deprecations

**Comment by stof** [comment]
you are missing the installation of dependencies

File: composer.json (hunk 1 above)
Comment (aitboudad) [inline composer.json]: done

**Comment by willdurand** [comment]
thanks!

File: .travis.yml [hunk 2]
```diff
@@ -17,15 +25,11 @@ script:
     - phantomjs Resources/js/run-qunit.js file://`pwd`/Resources/js/index-with-es5-shim.html
 
 matrix:
-    allow_failures:
-        - env: SYMFONY_VERSION=dev-master
-        - php: hhvm
+    fast_finish: true
     include:
+      - php: 5.3.3
+        env: COMPOSER_FLAGS='--prefer-lowest --prefer-stable'
```
Comment (stof) [inline .travis.yml]: this does not work, as you don't use this variable

Comment (willdurand) [inline .travis.yml]: ah. I suppose it is not an env var Composer would read by itself then?

Comment (willdurand) [inline .travis.yml]: those options are available on the `update` command only anyway, so removing them.

**Pull request merged**